spot-vaults/contracts/charm/UsdcSpotManager.sol

```diff
@@ -144,8 +168,55 @@ contract UsdcSpotManager is Ownable {
         return VAULT.isUnderweightToken0();
     }
 
+    /// @notice Calculates the Univ3 tick equivalent of the given deviation factor.
+    function deviationToTicks(uint256 deviation) public pure returns (int24) {
+        // 2% ~ 200 ticks -> (POOL.tickSpacing())
+        // NOTE: width can't be zero, we set the minimum possible to 200.
+        uint256 t = deviation.mulDiv(10000, ONE);
+        t -= (t % 200);
+        return (t >= 200 ? SafeCast.toInt24(SafeCast.toInt256(t)) : int24(200));
+    }
+
     /// @return Number of decimals representing 1.0.
     function decimals() external pure returns (uint8) {
         return uint8(DECIMALS);
     }
+
+    /// @dev Configures the vault to provide concentrated liquidity in the active zone.
+    function _setupActiveZoneLiq(uint256 deviation) private {
+        (uint256 usdcBal, ) = VAULT.getTotalAmounts();
+        uint256 fullRangeWeight = uint256(VAULT_MAX_FRW).mulDiv(
+            fullRangeMaxUsdcBal,
+            usdcBal
+        );
+        uint256 limitBandDeviationWidth = isOverweightSpot()
+            ? activeZoneDeviation.upper - deviation
+            : deviation - activeZoneDeviation.lower;
+        VAULT.setFullRangeWeight(SafeCast.toUint24(fullRangeWeight));
+
+        // IMPORTANT: The deviation percentage, to ticks conversion isn't precise.
+        // In the worst case, there might be liquidity up to 2% outside the active zone definition.
+        VAULT.setBaseThreshold(deviationToTicks(concBandDeviationWidth));
+        VAULT.setLimitThreshold(deviationToTicks(limitBandDeviationWidth));
```

Comment:
We can consider removing the limit ranges completely when it's buying above fmv and selling below. It does slightly reduce the opportunity of getting back to 50/50 in the long run, however it also eliminates IL. 
But considering the active zone is quite tight to begin with maybe it doesn't matter.

Reply:
If we can, we should have the logic make sense independent of configuration. At this stage, it's best if we don't make any assumptions about how tight the active zone is

Reply:
Right, makes sense to remove it then.

Reply:
Okay, will go back to adding limit ranges only based on fmv

Reply:
Okay, rethinking this framing re IL. I think the original implementation makes the most sense. We keep concentrated/limit liquidity when in the active zone and revert to bounded infinite band when outside.